requirements.txt

```diff
@@ -41,3 +41,4 @@ deprecated~=1.2
 jinja2~=3.1, >=3.1.3
 anyio~=3.7
 orjson~=3.9
+python-multipart~=0.0.6
```

Comment:
Not sure what uses it?

Reply:
@liranbg It was needed for old version of PR. I've removed it.